coalib/collecting/Collectors.py

```diff
@@ -369,6 +369,10 @@ def get_all_bears_names():
     return [bear.name for bear in get_all_bears()]
 
 
+def _argcomplete_bears_names(*args, **kwargs):
```

Comment:
One question does `*args, **kwargs` is needed here?
I don't see it getting anywhere used. or you used it for future references?

Reply:
It is called by `argcomplete`.

See `EnvironCompleter` in https://github.com/kislyuk/argcomplete#specifying-completers